src/nvim/lib/kbtree.h

```diff
@@ -0,0 +1,396 @@
+#ifndef __AC_KBTREE_H
```

Comment:
I'm not the lawyer but it looks illegal to use this source code without the original [copyright](https://github.com/attractivechaos/klib/blob/master/kbtree.h#L1-L26). For example [khash.h](https://github.com/neovim/neovim/blob/master/src/nvim/lib/khash.h), [klist.h](https://github.com/neovim/neovim/blob/master/src/nvim/lib/klist.h) and [kvec.h](https://github.com/neovim/neovim/blob/master/src/nvim/lib/kvec.h) have their respective copyrights.